src/main/java/bio/terra/cli/command/workspace/Create.java

```diff
@@ -18,14 +19,19 @@ public class Create extends BaseCommand {
 
   @CommandLine.Mixin WorkspaceNameAndDescription workspaceNameAndDescription;
 
+  @CommandLine.Mixin WorkspaceProperties workspaceProperty;
```

Comment:
workspaceProperty -> workspaceProperties

Since class is `WorkspaceProperties`